external/netfx/netfx.depproj

```diff
@@ -13,10 +13,16 @@
   <PropertyGroup Condition="!$(NuGetTargetMoniker.StartsWith('.NETFramework'))">
     <!-- For things not .NETFramework we need the net461 targeting pack to generate facades -->
     <NuGetTargetMoniker>.NETFramework,Version=v4.6.1</NuGetTargetMoniker>
+    <NuGetTargetMonikerShort>net461</NuGetTargetMonikerShort>
```

Comment:
It gets consumed here in [packageresolve.targets](https://github.com/dotnet/corefx/pull/18035/files#diff-3045b99d7cac6dab2d9cddaf1b3cbc2fR40). Unfortunately, passing the "full moniker" to `/p:TargetFramework` does not work with the 2.0 CLI / SDK.